opal/datatype/opal_convertor.c

```diff
@@ -40,18 +40,33 @@
 #include "opal/datatype/opal_datatype_checksum.h"
 #include "opal/datatype/opal_datatype_internal.h"
 #include "opal/datatype/opal_datatype_prototypes.h"
-#if OPAL_CUDA_SUPPORT
-#    include "opal/cuda/common_cuda.h"
-#    define MEMCPY_GPU(DST, SRC, BLENGTH, CONVERTOR) \
+#include "opal/mca/accelerator/accelerator.h"
+#    define MEMCPY_ACCELERATOR(DST, SRC, BLENGTH, CONVERTOR) \
```

Comment:
blank line between the includes and the define.  Also, with the #ifdef removed, there shouldn't be spacing between the `#` and `define`.

Reply:
Cleaned up.